themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl

```diff
@@ -25,11 +25,16 @@
 
 <div id="search_filters_brands">
   <section class="facet">
-    <h1 class="h6 text-uppercase facet-label">
-      {if $display_link_brand}<a href="{$page_link}" title="{l s='Brands' d='Shop.Theme.Catalog'}">{/if}
-        {l s='Brands' d='Shop.Theme.Catalog'}
-      {if $display_link_brand}</a>{/if}
-    </h1>
+    {if $display_link_brand}
+      <a href="{$page_link}" class="h6 text-uppercase facet-label" title="{l s='brands' d='shop.theme.catalog'}">
+        {l s='brands' d='shop.theme.catalog'}
+      </a>
+    {else}
+      <p class="h6 text-uppercase facet-label">
+        {l s='brands' d='shop.theme.catalog'}
```

Comment:
```suggestion
        {l s='Brands' d='shop.theme.catalog'}
```
It seems you changed that

Reply:
Damn, what happened, it was a copy pasta